pkg/controller/sync/resource.go

```diff
@@ -158,45 +155,29 @@ func (r *federatedResource) SkipClusterChange(clusterObj pkgruntime.Object) bool
 
 // TODO(marun) Marshall the template once per reconcile, not per-cluster
 func (r *federatedResource) ObjectForCluster(clusterName string) (*unstructured.Unstructured, error) {
-	// Federation of namespaces uses Namespace resources as the
-	// template for resource creation in member clusters. All other
-	// federated types rely on a template type distinct from the
-	// target type.
-	//
-	// Namespace is the only type that can contain other resources,
```

Comment:
Doesn't this comment still hold good?

Reply:
The comment still holds, but it does not belong in this function.  I've added a note to our technical writer to have mention of it added to the documentation.